### ISSUE TITLE
Upload an up to date release?

### ISSUE THREAD

**Issue by Vulldozer** [ariccio/altWinDirStat#22]
The latest release is from 2015 so I tried compiling the program on my own and failed miserably, can we get an updated version in the github releases please.

**Comment by Luckz** [comment]
(If it's just about using a fast directory visualization tool, Treesize Free and Wiztree both provide that, and both can be installed/updated via scoop.)

**Comment by ariccio** [comment]
Yes, this is a hobby project that I work on, but not enough :)

Because of a change in the way the MSVC C++ implementation of std::async works, performance is *worse* right now until I completely rewrite the whole parallel directory tree enumeration mechanism!

`master` should compile for you in VS2019 right now.

**Comment by ariccio** [comment]
Lately, I've been putting a lot more dev work into citizen science efforts for the COVID response, and trying to get my career going! I really *should* clean things up a bit and release a new version!

**Comment by Vulldozer** [comment]
Thank you a lot for your effort

**Comment by ariccio** [comment]
Yes, Wiztree is free and works quite well too! IIRC though it requires admin permissions to achieve that speed?

In the mean time I might give USN enumeration another go - I think I can make that work well this time. I should still rewrite the async mechanism so you get fast analysis with a standard user account.
